bitrise.yml

```diff
@@ -1475,13 +1475,13 @@ meta:
 
 trigger_map:
 - push_branch: main
-  pipeline: pipeline_multiple_shards
+  workflow: perfherder-test
```

Comment:
I think we can undo this change, tests are passing and the PERFHERDER_DATA object looks fine,  @jjSDET is there a way to confirm it is correct?

Reply:
![Screenshot 2023-06-29 at 11 26 42 AM](https://github.com/mozilla-mobile/firefox-ios/assets/107960801/88dca0c7-61f6-4ec1-b78a-c41ce7ec799d)

Reply:
for comparison, you can see all the data is formatted the same as with the previous perfherder data